ingestion_tools/scripts/importers/key_image.py

```diff
@@ -69,27 +70,53 @@ def get_existing_preview(self) -> tuple[np.ndarray | None, int | None]:
             return data, data.shape[-1]
         return None, None
 
-    def load_annotations(self) -> Generator[np.ndarray, None, None]:
+    def load_point_annotations(self) -> Generator[KeyPointAnnotation, None, None]:
+        vis_config_importer = VisualizationConfigImporter(self.config, {}, parents=self.parents)
         for annotation in AnnotationImporter.finder(self.config, **self.parents):
```

Comment:
This will only fetch the annotations in the config.

Reply:
I think that was the way it worked before as well.

Reply:
Yes, it is. I called it out as there was an offline discussion about including annotations that aren't a part of the deposition in the key image. :)